src/i18n/en-US.properties

```diff
@@ -96,6 +96,10 @@ error_not_downloadable=Oops! It looks like something is wrong with this file. We
 error_flash_not_enabled=We’re sorry, the preview didn’t load because Flash is not enabled on your browser. If possible, please enable Flash and refresh the page.
 # Preview error message shown when loading what would be an Office Excel Online file extension in Internet Explorer
 error_internet_explorer_office_online=Microsoft 365 apps and services no longer support Internet Explorer 11, therefore, Box users may have a degraded experience.
+# Preview error message shown when the preview fails because the file is too large for its extension type
+error_large_file=File exceeds supported size for preview. Open or download the file to view.
```

Comment:
Minor, but the language and tone of these messages isn't consistent with the existing errors. Something like the following would work:

```
We're sorry, the file size is too large to preview. Please open or download the file to view it.